### PR TITLE
Add links to glitch.me template using polyserve. Fixes #5016

### LINKED ISSUE #5016: Provide a template for reporting IE11 issues
Creating jsfiddle/jsbin for IE11 isn't possible, but you still asking for it in ISSUE_TEMPLATE.md

I wonder if there any recommended by the Polymer team way to provide live demos for reporting IE11 issues?

### PR DESCRIPTION
**Pull request by kevinpschaaf**
Use glitch.me template instead of jsBin, so that it can be served with polyserve for transpilation of ES6 on older browsers.

### Reference Issue
Fixes #5016

### CODE REVIEW

**Comment by TimvdLippe** [comment]
Hm, the glitch website itself does not seem to load in IE11 🤔

**Comment by TimvdLippe** [comment]
Oh nvm. I can do "show live" and then open that link in IE11. Maybe worth mentioning in the guide as well?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -9,8 +9,11 @@ If you are asking a question rather than filing a bug, try one of these instead:
 <!-- Example: Error thrown when calling `appendChild` on Polymer element -->
 
 #### Live Demo
-<!-- Fork this JSBin, or provide your own URL -->
+<!-- jsBin starting point (fork and edit) -->
 http://jsbin.com/luhaxab/1/edit
+<!-- glitch.me starting point (remix and edit -- must be logged in to persist!) -->
```
Comment (limonte) [inline .github/ISSUE_TEMPLATE.md]: it's glitch.com

**Comment by limonte** [comment]
glitch.com works [perfectly](https://github.com/Polymer/polymer/issues/5015#issuecomment-356585371) for me :+1:

**Comment by limonte** [comment]
I have the bad news as well. glitch.com doesn't work well as I mention in my previous comment.

For this project https://glitch.com/edit/#!/hissing-surf the live demo is returning **504 Gateway Time-out** without any additional information: https://hissing-surf.glitch.me/

**Comment by TimvdLippe** [comment]
Hm it is seems to be working for me. It might have been a time bug in the matrix?

**Pull request merged**